save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/mobile/WelcomeMobileView.kt

```diff
@@ -29,6 +33,13 @@ class WelcomeMobileView : AbstractView<WelcomeProps, IndexViewState>(false) {
                 background =
                         "-webkit-linear-gradient(270deg, rgb(209, 229, 235),  rgb(217, 215, 235))".unsafeCast<Background>()
             }
+            // FixMe: Note that they block user interactions. Particles are superimposed on top of the view in some transitions
+            // https://github.com/matteobruni/tsparticles/discussions/4489
+            Particles::class.react {
```

Comment:
Does it work fine on mobile devices as well?

Reply:
Yeah, I was surprised